core/trino-main/src/main/java/io/trino/metadata/DisabledSystemSecurityMetadata.java

```diff
@@ -195,6 +195,9 @@ public Optional<Identity> getFunctionRunAsIdentity(Session session, CatalogSchem
         return Optional.empty();
     }
 
+    @Override
+    public void functionCreated(Session session, CatalogSchemaFunctionName function) {}
```

Comment:
Please also add `functionDropped` and `functionRenamed` if `ALTER FUNCTION RENAME` exists.

Reply:
There is no ALTER FUNCTION